### PR TITLE
add timeouts to queries used for login

### PR DESCRIPTION
**Pull request by DylanVolz**

### CODE REVIEW

File: traffic_ops/traffic_ops_golang/auth/login.go [hunk 1]
```diff
@@ -107,7 +108,10 @@ func LoginHandler(db *sqlx.DB, cfg config.Config) http.HandlerFunc {
 
 func CheckLocalUserIsAllowed(form passwordForm, db *sqlx.DB) (bool, error) {
 	var roleName string
-	err := db.Get(&roleName, "SELECT role.name FROM role INNER JOIN tm_user ON tm_user.role = role.id where username=$1", form.Username)
+	dbCtx, dbClose := context.WithTimeout(context.Background(), time.Second*20)
```
Comment (rob05c) [inline traffic_ops/traffic_ops_golang/auth/login.go]: Can you put this in a config? It'd be super-helpful for debugging to be able to change it dynamically in an installation, without a new binary/RPM.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2210/
Test PASSed.

File: traffic_ops/traffic_ops_golang/auth/login.go (hunk 1 above)
Comment (DylanVolz) [reply traffic_ops/traffic_ops_golang/auth/login.go]: +1, I will update this PR.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2219/
Test PASSed.

**Comment by rob05c** [comment]
Tested, everything seems to work.

**Pull request merged**